static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -427,6 +440,24 @@ class UsageStatsOrganization<
             return;
           }
 
+          if (outcome === Outcome.CLIENT_DISCARD) {
+            usageStats[i].discarded += stat;
```

Comment:
Nit, but due to consistency with other part of code inside the function maybe we could also use: `usageStats[i][outcome] += stat;` here.

On the next refactor Friday, we can refactor this function then, cause it seems to me that it can be written more concise, but that is outside of the scope of this PR :)

Reply:
you have good eyes :) but we cannot do that, because we get 'client_discarded' from the backend and in the frontend we are naming it 'discarded'

Reply:
the function can for sure be improved but as you said it is outside of the scope of this PR  - maybe in a follow-up

Reply:
My bad, I missed that, makes sense now why are we using it like this, thanks :)